hazelcast/src/main/java/com/hazelcast/core/Member.java

```diff
@@ -258,4 +260,15 @@
      */
     void removeAttribute(String key);
 
+    /**
+     * Returns the Hazelcast codebase version of this member; this may or may not be different from the version reported by
+     * {@link ClusterService#getClusterVersion()}, for example when a node with a different codebase version is added to an
+     * existing cluster. See the documentation for {@link ClusterService#getClusterVersion()} for a more thorough discussion
+     * of {@code Cluster} and {@code Member} / {@code Node} version.
```

Comment:
Can you change `{@link ClusterService#getClusterVersion()}` to `{@link Cluster#getClusterVersion()}`? This is a public class and I don't think it's desired to refer/import an internal class here.